packages/RichTextEditing-Tests.package/RichTextEditorImportExportTests.class/instance/assert.equals.withExport..st

```diff
@@ -0,0 +1,8 @@
+utilities
+assert: aString equals: aRichTextDocument withExport: aRichTextExportStrategy
+
+	| fileStream |
+	aRichTextExportStrategy export: aRichTextDocument to: self exampleExportFilePath.
+	fileStream := StandardFileStream readOnlyFileNamed: self exampleExportFilePath.
```

Comment:
Maybe we should mock the file

Reply:
Possible. But I think there should also be tests that explicitly test writing on disk

Reply:
Maybe it's not appropriate to mock the FileStream since the tested classes don't actually interact with a FileStream and only with a file path, which is not useful to mock.

Reply:
That's true. How about introducing something like `export: aDocument on: aStream` and call that one inside the original `export:to:` to allow testing the export seperate from file system logic?